unittests/InterOp/ScopeReflectionTest.cpp

```diff
@@ -544,6 +544,7 @@ TEST(ScopeReflectionTest, GetBaseClassOffset) {
     class B { int b; };
     class C : public A, public B { int c; };
     class D : public A, public B, public C { int d; };
+    class E : virtual A { int e; };
```

Comment:
Our test should include at least two base classes with a common base. Then in the check we should ensure that there is only one base.

Reply:
Perhaps the case `D` above is better to check.

Reply:
```c++
class F : virtual A { int f; }
class G : public F, public E { int g; }
```
Like this?